src/constants/testInfo.ts

```diff
@@ -0,0 +1,23 @@
+/*
+ *  Copyright 2023 EPAM Systems
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *  http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ *
+ */
+
+export const RPTestInfo = {
```

Comment:
Mostly in this project we use `enum`s to define constants.
And one question - why don't you use the `EVENTS` defined in `client-javascript`?
These test info details are still the same as the reported via old one `ReportingAPI`.